Libraries/WebSocket/RCTReconnectingWebSocket.m

```diff
@@ -63,6 +84,9 @@ + (void)load
     rebind_symbols((struct rebinding[1]){
       {"nwlog_legacy_v", my_nwlog_legacy_v, (void *)&orig_nwlog_legacy_v}
     }, 1);
+    rebind_symbols((struct rebinding[1]){
+      {"__nwlog_pack", my__nwlog_pack, (void *)&orig__nwlog_pack}},
+                   1);
```

Comment:
I believe you need to "fix" the indentation here.

Reply:
Fixed